ci/azure-windows-36.yaml

```diff
@@ -5,12 +5,14 @@ channels:
 dependencies:
   - blosc
   - bottleneck
+  - boost-cpp<1.67
   - fastparquet
   - feather-format
   - matplotlib
   - numexpr
   - numpy=1.14*
   - openpyxl=2.5.5
+  - parquet-cpp
   - pyarrow
   - pytables
```

Comment:
Could you try `vc=14.0` (without the pragma stuff)? the last working build had that, while the newer ones have `14.1`. Plus, the `notna` definition still didn't fix the errors in the WinPy36 build.

Reply:
> the notna definition still didn't fix the errors in the WinPy36 build.
 
29f392c had a bug (`not` isn't valid C++).

I think this turned up a real bug that's hitting users / will hit users, depending on how things are compiled. Probably best to fix it.